cpu/saml21/cpu.c

```diff
@@ -82,6 +82,28 @@ uint32_t sam0_gclk_freq(uint8_t id)
     }
 }
 
+void cpu_pm_cb_enter(int deep)
+{
+    if (deep) {
```

Comment:
`deep` is always `0`. I think you should check `PM->SLEEPCFG.bit.SLEEPMODE`.

Reply:
Good catch - maybe we should just set deep for consistency.